### ISSUE TITLE
Not a WSDL

### ISSUE THREAD

**Issue by riramar** [NetSPI/Wsdler#5]
Just installed Wsdler from BApp Store and following the steps from here https://blog.netspi.com/hacking-web-services-with-burp/ I'm always getting the "Parsing WSDL" and after that the error "Not a WSDL". The same WSDL can be parsed by SoapUI without any problem.
Is there a log or debug mode?

**Comment by egru** [comment]
What's the MIME type on the WSDL response? It will only parse the WSDL if it's XML.

**Comment by riramar** [comment]
I'll check and let you know.

**Comment by riramar** [comment]
You're right. The server is returning "Content-Type: text/html;charset=UTF-8".
I'm not the system admin neither a developer for this system.
Can I change this header and ask Wsdler to parse?

**Comment by riramar** [comment]
I gave a try but it didn't work. Just changed to "Content-Type: text/xml" but when I tried to parse from edited response on the Burp History I got the same error.

**Comment by egru** [comment]
I created a new jar that removes the MIME check. You can grab it here: https://github.com/NetSPI/Wsdler/releases/tag/2.0.11.1

**Comment by riramar** [comment]
It worked! :)
Thanks!

**Comment by prvnkumark** [comment]
Hi,

I added wsdlr version 2.0.11 from extensions, however wsdler is not parsing the wsdl file. Do i have to run via command line? 

SOAP UI is parsing the wsdl without any issues.

**Comment by egru** [comment]
I'll take a look at it. The MIME check is removed from the initial look at the response, but there could be a issue further down in the parsing libraries. Is there any other error message that you're getting?

**Comment by prvnkumark** [comment]
Hi,

I am not getting any other error message, have you got chance to look into it?
